packages/admin-client/src/CustomHeader.tsx

```diff
@@ -90,7 +91,26 @@ const Root2 = styled(AppBar, {
 })
 
 const defaultMenu = <HorizontalMenu />
-const defaultUserMenu = <UserMenu />
+
+const CustomUserMenu = () => {
+    const logout = useLogout()
+    return (
+        <UserMenu>
+            <MenuItem
+                onClick={() => {
+                    localStorage.removeItem("token")
+                    localStorage.removeItem("refresh_token")
+                    localStorage.removeItem("permissions")
+                    localStorage.removeItem("access")
+                    localStorage.clear()
+                    logout()
+                }}
+            >
+                <ExitIcon /> Logout
+            </MenuItem>
+        </UserMenu>
+    )
+}
```

Comment:
I think we can do this without introducing the CustomUserMenu:

useLogout() invokes authProvider.logout(), which currently removes the first 3 items from localStorage. So those lines execute here, then again when logout() gets called.

I think if we add the additional lines to authProvider.logout(), it should have the desired effect whenever logout occurs, without requiring the CustomUserMenu?

Reply:
Removed the CustomuserMenu and switched to authProvider to handle cleanup after logout.